Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -158,6 +158,9 @@ function runHelm(helmCli: helmcli, command: string, kubectlCli: kubernetescli, f
             tl.warning("Capturing deployment metadata failed with error: " + e);
         }
     }
+    else {
+        tl.setVariable('helmExitCode', execResult.stdout.toString());
```

Comment:
You're updating a wrong variable here

Reply:
Oops, corrected that and moved the code further up as suggested.